homework.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+using Newtonsoft.Json;
+
+namespace Notino.Homework
+{
+    public class Document
+    {
+        public string Title { get; set; }
+        public string Text { get; set; }
+    }
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var sourceFileName = Path.Combine(Environment.CurrentDirectory, "..\\..\\..\\Source Files\\Document1.xml");
+            var targetFileName = Path.Combine(Environment.CurrentDirectory, "..\\..\\..\\Target Files\\Document1.json");
+
+            try
+            {
+                FileStream sourceStream = File.Open(sourceFileName, FileMode.Open);
+                var reader = new StreamReader(sourceStream);
+                string input = reader.ReadToEnd();
+            }
+            catch (Exception ex)
+            {
+                throw new Exception(ex.Message);
+            }
+
+            var xdoc = XDocument.Parse(input);
+            var doc = new Document
```

Comment:
pouzil bych asi nejaky parser co to vyparsuje podle typu xmlSerializer, kdyby to nevyhovovalo tak mozna pomoci reflexe navrhnout vlastni. Tedka je to asi ok, ale kdybych pridaval dalsi parametry mohl bych nekde zapomenou ho pridat a rozbije se to.